golang/pkg/crane/k8sconfig/secret.go

```diff
@@ -0,0 +1,97 @@
+package k8sconfig
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	config "github.com/dyrector-io/dyrectorio/golang/internal/config"
+	craneConfig "github.com/dyrector-io/dyrectorio/golang/pkg/crane/config"
+	"github.com/dyrector-io/dyrectorio/golang/pkg/crane/k8s"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+const SecretFileName = "private.key"
+
+func TryToGetValidSecret(ctx context.Context, namespace, name string, appConfig *craneConfig.Configuration) (string, error) {
+	if namespace == "" || name == "" {
+		return "", fmt.Errorf("secret name and namespace can't be empty")
```

Comment:
You are using OR operator in `if` so use it in the error message too.

Reply:
Oh you're right. I'll fix it.